pkg/scheduler/framework/plugins/noderesources/least_allocated.go

```diff
@@ -63,25 +64,58 @@ func (la *LeastAllocated) ScoreExtensions() framework.ScoreExtensions {
 }
 
 // NewLeastAllocated initializes a new plugin and returns it.
-func NewLeastAllocated(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error) {
+// resource without the weight defined or explicitly defined as zero will be applied with defined weight, i.e. "1" instead,
```

Comment:
this is not the API godoc :)

See my other comment.

Reply:
thanks for fixing my mistake!